public/js/group/create.js

```diff
@@ -44,8 +44,28 @@ const addGroup = (event) => {
   }
 }
 
+const editGroup = (event) => {
+  event.preventDefault()
+  const formData = new FormData(event.target)
+  const errors = validateGroup(formData)
+
+  if (errors.length) {
+    clearMessages()
+    errors.forEach((err) => displayMessage('danger', err))
+  } else {
+    fetch(`/groups/${groupId}`, { method: 'POST', body: formData })
```

Comment:
For editing, it's better to use PUT or PATH request. POST is meant for creating resources. [[link]](https://www.restapitutorial.com/lessons/httpmethods.html)